core/src/main/resources/org/apache/spark/ui/static/timeline-view.css

```diff
@@ -223,11 +223,10 @@ rect.getting-result-time-proportion {
 }
 
 .vis-timeline .vis-item.executor.vis-selected {
-  background-color: #A2FCC0;
-  border-color: #36F572;
-  z-index: 2;
+    background-color: #00aaff;
+    border-color: #184c66;
+    z-index: 2;
```

Comment:
Thank you for update, @TomokoKomiyama . 
Could you keep the original indentation? Please see the other lines in this file.

Reply:
Also, please update the PR description. So far, it's still outdated.

Reply:
Thank you for your review.
I've fixed indentation, and I'm going to update the PR one.